python/v0.6.1/generate_fixtures.py

```
@@ -0,0 +1,126 @@
+    print("\nGenerated elements:")
+    for element_type in ["images", "labels", "points", "shapes", "tables"]:
+        elements = getattr(sdata, element_type, None)
+        if elements:
+            # Handle both dict and list cases
+            if isinstance(elements, dict):
+                print(f"  - {element_type}: {len(elements)} element(s)")
+                for name in elements.keys():
+                    print(f"    * {name}")
+            elif isinstance(elements, list):
+                print(f"  - {element_type}: {len(elements)} element(s)")
+                for i, elem in enumerate(elements):
```

Ruff (0.14.14)

[warning] Loop control variable `elem` not used within loop body

Rename unused `elem` to `_elem`

(B007)